make/dev.mk

```diff
@@ -0,0 +1,72 @@
+MINISHIFT_IP?=$(shell minishift ip)
+MINISHIFT_HOSTNAME=minishift.local
+MINISHIFT_HOSTNAME_REGEX='minishift\.local'
+ETC_HOSTS=/etc/hosts
+
+# to watch all namespaces, keep namespace empty
+APP_NAMESPACE ?= $(LOCAL_TEST_NAMESPACE)
+LOCAL_TEST_NAMESPACE ?= "toolchain-host-operator"
```

Comment:
Do we want to run it in the same namespace where we are running host operator?

Reply:
It was my intention. In prod we will run reg service in the host operator namespace. So I thought it would make sense to do the same in dev env.

Reply:
I do agree that, we need to run this service in the same namespace where we'll be running host-operator, but then we need to use the same here - https://github.com/codeready-toolchain/registration-service/pull/36/files#r329978845

Reply:
I'm going to remove deployment_dev_cluster.yaml from this PR :)